modules/renter/hostdb/scan.go

```diff
@@ -254,6 +194,26 @@ func (hdb *HostDB) threadedProbeHosts() {
 		case <-hdb.tg.StopChan():
 			return
 		case hostEntry := <-hdb.scanPool:
+			// Block the scan until the host is online.
```

Comment:
what happens if the hostdb is deleted and a full blockchain rescan occurs without internet? Do the host announcements accumulate in a queue somewhere, or do they get dropped?